spec/controllers/news_controller_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rails_helper'
+
+RSpec.describe NewsController, type: :controller do
+  describe 'GET #index' do
+    before do
+      3.times do |i|
+        create(:news, author: "Author #{i}", title: "Title #{i}", desc: "Desc #{i}", source: "http://example#{i}.com/", tech: 'ruby')
+      end
+      create(:news, author: "Author 3", title: "Title 3", desc: "Desc 3", source: "http://example3.com/", tech: 'rust')
```

Comment:
INFO: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)